pkg/projects/resource_project_test.go

```diff
@@ -126,7 +126,7 @@ func TestAccProjectInvalidDisplayName(t *testing.T) {
 
 func TestAccProject(t *testing.T) {
 	name := fmt.Sprintf("tftestprojects%s", randSeq(10))
-	resourceName := fmt.Sprintf("project.%s", name)
+	resourceName := fmt.Sprintf("project_project.%s", name)
 
 	params := map[string]interface{}{
 		"max_storage_in_gigabytes":   rand.Intn(100),
```

Comment:
unless you set the seed, rand isn't rand. So, for this case, it probably is fine because this number won't stick/matter. But you can't use it for resource creation

Reply:
Yep, agree